build/Microsoft.Toolkit.Uwp.nuspec

```diff
@@ -14,6 +14,11 @@
     <releaseNotes>First public release.</releaseNotes>
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
     <tags>UWP Toolkit Windows</tags>
+    <dependencies>
+      <group>
+        <dependency id="Newtonsoft.Json" version="[9.0.1,)" />
```

Comment:
What do you mean ? This is a notation to say I only take package with a version >= 9.0.1
If you want the specific version to be 9.0.1 only, I can make the change.